src/components/ArticleVotes.jsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from "react";
+import { updateArticleVotes } from "../api"
+
+const ArticleVotes = ({ article_id, votes_count }) => {
+    const [userVotes, setUserVotes] = useState(0);
+    const [isError, setIsError] = useState(false)
+
+    const handleLikeClick = (event) => {
```

Comment:
These two functions look great and seem to be handling the optimistic rendering really well. Could we find a way of refactoring to make it more DRY? The only thing that is different is whether we go up by 1 or down by 1?